cypress/support/index.js

```diff
@@ -1,19 +1 @@
 require('@mate-academy/cypress-tools/support');
```

Comment:
This file was reduced to one line, removing the custom Cypress command 'byDataCy'. This command was potentially being used in the test files. Removing it might break the tests. Make sure to update test files accordingly to avoid referencing this command.